A14-channelz.md

```diff
@@ -728,8 +730,27 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+message ChannelTraceEvent {
+  string description = 1;
+  string error = 2;
+  google.protobuf.Timestamp event_timestamp = 3;
+  ChannelConnectivityState state = 4;
+  // uuid of referenced subchannel.
+  // Optional, only present if this event refers to a child object.
+  string child_uuid = 5; 
+}
+
+message ChannelTraceData {
+  string uuid = 1;
+  int64 num_events_logged = 2;
+  google.protobuf.Timestamp channel_created_timestamp = 3;
+  repeated ChannelTraceEvent events = 4;
+}
+
 message ChannelTrace {
-  // TODO: fill this in.
+  ChannelTraceData channel_data = 1;
```

Comment:
This should have a ChannelRef or Subchannel ref back to which channel it came from.

Reply:
Ok. ChannelTrace can be for a channel or subchannel, so I included both, and stipulated that only one may be set